private/model/api/shape_value_builder.go

```diff
@@ -3,7 +3,9 @@
 package api
 
 import (
+	"github.com/aws/aws-sdk-go/aws"
```

Comment:
Nit should be using `goimports` to format the import statements with standard libary imports first, new line, followed by non-standard library imports.